lib/logstash-logger/formatter/base.rb

```diff
@@ -25,7 +25,9 @@ def build_event(message, severity, time)
                   when LogStash::Event
                     data.clone
                   when Hash
-                    event_data = data.merge("@timestamp".freeze => time)
+                    event_data = data.clone
+                    event_data['message'.freeze] = event_data.delete(:message) if event_data.key?(:message)
```

Comment:
Yep. I missed this optimisation somehow. Here is the proof (at least for `ruby 2.2.6p396 (2016-11-15 revision 56800)`):
```ruby
irb(main):001:0> "asd".object_id
=> 70178538223500
irb(main):002:0> "asd".object_id
=> 70178538189060
irb(main):003:0> "asd".freeze.object_id
=> 70178538223620
irb(main):004:0> "asd".freeze.object_id
=> 70178538223620
```